src/style.css

```diff
@@ -1 +1,51 @@
 /* add styles here */
+
+.page {
+  margin: 0;
+  padding: 0;
+  background-color: #fff;
+}
+
+.stars__box {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
The properties `background-repeat: no-repeat;` and `background-position: center;` are repeated here and in the `.stars__star` class. You can remove them from here since they are already defined in the `.stars__star` class.